src/commands/link.ts

```diff
@@ -20,7 +20,10 @@ import { AnnounceService } from '../service/AnnounceService';
 import { CommandUtils } from '../service/CommandUtils';
 
 export default class Link extends Command {
-    static description = `It announces VRF and Voting Link transactions to the network for each node with 'Peer' or 'Voting' roles. This command finalizes the node registration to an existing network.`;
+    static description = `It announces VRF and Voting Link transactions to the network for each node with 'Peer' or 'Voting' roles. This command finalizes the node registration to an existing network.
+
+This is an ONLINE command as it creates the transactions depending on the main account type (simple or multisig), the main account link status, and it announces the transactions to the network.
```

Comment:
```suggestion
This command can only be used in ONLINE mode since it creates the transactions depending on the main account type (simple or multisig), the main account link status, and it announces the transactions to the network.
```